### PR TITLE
CLAWdit

### PR DESCRIPTION
**Pull request by seth-shaw-unlv**
**GitHub Issue**: Islandora-CLAW/CLAW/issues/1267

[Tech Call minutes, item 1.](https://github.com/Islandora-CLAW/CLAW/wiki/September-25%2C-2019#minutes)

# What does this Pull Request do?

Updates the README to better conform to the standard template.

# What's new?

* Changes a few README links.
* Added missing sections.

# How should this be tested?

Give it a read and compare it to [the template](https://github.com/Islandora-CLAW/CLAW/blob/master/docs/contributing/readme_template.md).

# Additional Notes:
@manez is [looking into a module mascot](https://islandora.slack.com/archives/CM5PPAV28/p1569444637289100).

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

**Comment by manez** [comment]
Mascot!

image

**Comment by seth-shaw-unlv** [comment]
I was just going to ask for that! Thanks.

**Comment by seth-shaw-unlv** [comment]
Ack! You approved it @manez and then I made changes. Sorry!

**Comment by seth-shaw-unlv** [comment]
Great! Although I presume we still need a @Islandora-CLAW/committers to review/merge it.

**Comment by manez** [comment]
Indeed. I'm just giving my non-Committer thumbs up 👍

File: README.md [hunk 1]
```diff
@@ -102,3 +121,32 @@ can change, for example, the separator and the date order to display dates in
...
+
+* [Seth Shaw](https://github.com/seth-shaw-unlv)
+
+## Development
+
+If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
+
+If you would like to contribute code to the project, you need to be covered by an Islandora Foundation [Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_cla.pdf) or [Corporate Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_ccla.pdf). Please see the [Contributors](http://islandora.ca/resources/contributors) pages on Islandora.ca for more information.
+
+We recommend using the [claw-playbook](https://github.com/Islandora-Devops/claw-playbook) to get started. If you want to pull down the submodules for development, don't forget to run `git submodule update --init --recursive` after cloning.
```
Comment (dannylamb) [inline README.md]: You can delete this

> If you want to pull down the submodules for development, don't forget to run `git submodule update --init --recursive` after cloning.

Comment (seth-shaw-unlv) [reply README.md]: Done.

**Pull request merged**